3rd_party/requests/README.md

```diff
@@ -0,0 +1,29 @@
+# Requests -library
+
+Requests -library is one of the most known and used Python libraries.
+
+👉 Their slogan tells it all: `HTTP for Humans`
+
+Whether you need to download a file or check if a website is responding, for everything HTTP, you should just use `requests.`
+
+## Usage
+
+```python
+import requests
+
+def download_file(url, local_filename):
+    response = requests.get(url)
+    response.raise_for_status()  # This will raise an exception if the request fails
+    with open(local_filename, 'wb') as f:
+        f.write(response.content)  # Write the content of the response to a file
+    return local_filename
+```
+
+Examples in Robocorp Portal:
+* [Python PLaywright -template](https://robocorp.com/portal/robot/robocorp/template-python-browser)
```

Comment:
This example is almost right, as it needs to be refactored in order to use the `requests` library instead.

Reply:
Done with [PR](https://github.com/robocorp/template-python-browser/pull/3).